ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.ts

```diff
@@ -308,6 +309,11 @@ export class ParticipantListComponent implements OnInit {
                   this.sourceColumns[ 'p' ] = [];
                 }
                 this.sourceColumns[ 'p' ].push(filter);
+              } else if (key === 'sm') {
+                if (this.sourceColumns[ 't' ] == null || this.sourceColumns[ 't' ] == undefined) {
```

Comment:
Can be shortened as 
`if (this.sourceColumns[ 't' ] == null) {...}`

`==` matches `null` and `undefined`